docs/rook.md

```diff
@@ -0,0 +1,212 @@
+#     }
+#   },
+#   "conditions": [
+#     {
+#       "lastHeartbeatTime": "2022-09-14T08:57:02Z",
+#       "lastTransitionTime": "2022-09-14T08:56:59Z",
+#       "message": "Cluster created successfully",
+#       "reason": "ClusterCreated",
+#       "status": "True",
+#       "type": "Ready"
+#     },
+#     {
+#       "lastHeartbeatTime": "2022-09-14T08:57:30Z",
+#       "lastTransitionTime": "2022-09-14T08:57:30Z",
+#       "message": "Detecting Ceph version",
+#       "reason": "ClusterProgressing",
+#       "status": "True",
+#       "type": "Progressing"
+#     }
+#   ],
+#   "message": "Detecting Ceph version",
+#   "observedGeneration": 2,
+#   "phase": "Progressing",
+#   "state": "Creating",
+#   "version": {
+#     "image": "quay.io/ceph/ceph:v17",
+#     "version": "17.2.3-0"
+#   }
+```
+
+## Status All
+
+```bash
+kubectl rook-ceph rook status all
+
+# cephblockpoolradosnamespaces.ceph.rook.io:
+# cephblockpools.ceph.rook.io: {
+#   "observedGeneration": 2,
+#   "phase": "Failure"
+# }
+# {
+#   "phase": "Progressing"
+# }
+# cephbucketnotifications.ceph.rook.io:
+# cephbuckettopics.ceph.rook.io:
+# cephclients.ceph.rook.io:
+# cephclusters.ceph.rook.io: {
+#   "ceph": {
+#     "capacity": {},
+#     "fsid": "b74c18dd-6ee3-44fe-90b5-ed12feac46a4",
+#     "health": "HEALTH_OK",
+#     "lastChecked": "2022-09-14T08:57:00Z",
+#     "versions": {
+#       "mgr": {
+#         "ceph version 17.2.3 (dff484dfc9e19a9819f375586300b3b79d80034d) quincy (stable)": 1
+#       },
+#       "mon": {
+#         "ceph version 17.2.3 (dff484dfc9e19a9819f375586300b3b79d80034d) quincy (stable)": 3
+#       },
+#       "overall": {
+#         "ceph version 17.2.3 (dff484dfc9e19a9819f375586300b3b79d80034d) quincy (stable)": 4
+#       }
+#     }
+#   },
+#   "conditions": [
+#     {
+#       "lastHeartbeatTime": "2022-09-14T08:57:02Z",
+#       "lastTransitionTime": "2022-09-14T08:56:59Z",
+#       "message": "Cluster created successfully",
+#       "reason": "ClusterCreated",
+#       "status": "True",
+#       "type": "Ready"
+#     },
+#     {
+#       "lastHeartbeatTime": "2022-09-14T08:57:37Z",
+#       "lastTransitionTime": "2022-09-14T08:57:37Z",
+#       "message": "Configuring Ceph Mons",
+#       "reason": "ClusterProgressing",
+#       "status": "True",
+#       "type": "Progressing"
+#     }
+#   ],
+#   "message": "Configuring Ceph Mons",
+#   "observedGeneration": 2,
+#   "phase": "Progressing",
+#   "state": "Creating",
+#   "version": {
+#     "image": "quay.io/ceph/ceph:v17",
+#     "version": "17.2.3-0"
+#   }
+# }
+# cephfilesystemmirrors.ceph.rook.io:
+# cephfilesystems.ceph.rook.io:
+# cephfilesystemsubvolumegroups.ceph.rook.io:
+# cephnfses.ceph.rook.io:
+# cephobjectrealms.ceph.rook.io:
+# cephobjectstores.ceph.rook.io:
+# cephobjectstoreusers.ceph.rook.io:
+# cephobjectzonegroups.ceph.rook.io:
+# cephobjectzones.ceph.rook.io:
+# cephrbdmirrors.ceph.rook.io:
+# objectbucketclaims.objectbucket.io:
+```
+
+## Status CR-Name
```

Comment:
```suggestion
## CephCluster Status
```

Reply:
with these we have to pass cr name for which we want the status, so kept it like this